edx/edx-enterprise
# [ENT-752] Update DSC policy to match legal requirements.

This updates DSC policy language to match legal requirements.

**JIRA:** [ENT-752](https://openedx.atlassian.net/browse/ENT-752)

**Merge deadline:** End of sprint W.

**Testing instructions:**

1. Go to either the course or program enrollment landing page.
1. Click submit to get to the DSC page.
1. Observe the DSC language when expanding the dropdown, and make sure it matches new requirements.
